template/build/webpack.base.conf.js

```diff
@@ -41,6 +41,12 @@ module.exports = {
       },
       {{/lint}}
       {
+        test: /\.html/,
+        enforce: 'pre',
+        loader: 'htmlhint-loader',
+        exclude: /node_modules/
+      },
```

Comment:
I think this should be wrapped in `{{#lint}}...{{/lint}}` just like `eslint-loader` or have another key in `meta.js` something like `htmlLint`